config/coreboot-t430s.config

```diff
@@ -0,0 +1,286 @@
+#
```

Comment:
Please generate this config with
cp  config/coreboot-t430s.config build/coreboot-ver/.config
cd build/coreboot-ver/
make savedefconfig
cp defconfig ../../config/coreboot-t430s.config
git add ../../config/coreboot-t430s.config

So that it contains only deviations from std coreboot config